OpenTabletDriver.Configurations/Configurations/Huion/H950P.json

```
@@ -18,6 +18,21 @@
     "Touch": null
   },
   "DigitizerIdentifiers": [
+    {
+      "VendorID": 9580,
+      "ProductID": 100,
+      "InputReportLength": 12,
+      "OutputReportLength": null,
+      "ReportParser": "OpenTabletDriver.Configurations.Parsers.UCLogic.UCLogicTiltReportParser",
+      "FeatureInitReport": null,
+      "OutputInitReport": null,
+      "DeviceStrings": {
+        "201": "HUION_T(22d)_\\d{6}$"
```

This capturing group is meaningless.
```suggestion
        "201": "HUION_T22d_\\d{6}$"
```